mcodingbot/plugins/python.py

```diff
@@ -10,8 +10,9 @@
 PEP_REGEX = re.compile(r"pep *(?P<pep>\d{1,4})", re.IGNORECASE)
 
 
-def get_pep_link(pep_number: int) -> str:
-    return f"<https://peps.python.org/pep-{pep_number:04}/>"
+def get_pep_link(pep_number: int, hide_embed: bool) -> str:
```

Comment:
I prefer this signature for functions like this
```suggestion
def get_pep_link(pep_number: int, *, hide_embed: bool) -> str:
```

Reply:
boolean args should most of the time be keyword-only yeah